projects/jasn1-compiler/src/test/java-gen/org/openmuc/jasn1/compiler/rspdefinitions/ReplaceSessionKeysRequest.java

```diff
@@ -23,7 +23,7 @@
 import org.openmuc.jasn1.compiler.pkix1explicit88.Time;
 import org.openmuc.jasn1.compiler.pkix1implicit88.SubjectKeyIdentifier;
 
-public class ReplaceSessionKeysRequest implements BerType, Serializable {
+public class ReplaceSessionKeysRequest implements BerSerializable, BerType, Serializable {
```

Comment:
since BerSerializable extends BerType, is not it enough to extend only BerSerializable instead of doing both in this class and all other classes doing similar stuff?

Reply:
Hi @onderson.  Thanks for the feedback.  I'll look into it.